src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14357,37 @@ Label900: ;
 		return StringOut;
 	}
 
+	// adapted from http://stackoverflow.com/questions/4654636/how-to-determine-if-a-string-is-a-number-with-c
+	bool 
+	isNumber(std::string s)
+	{
+		std::string trimmed = stripped(s);
+		return(strspn(trimmed.c_str(), "-.0123456789eE") == trimmed.size());
+	}
+
+	// return the number of digits after the decimal point
+	// Glazer - November 2016
+	int
```

Comment:
So I was starting to draft a more efficient way to find the number of digits after the decimal, and then I realized this may just be an artifact of having to read the eio *back in*, so if we cleaned that process up so we were just storing the data in memory and writing it out directly without having to read the eio, this is probably all gone.